doc/api/process.markdown

```diff
@@ -237,13 +240,13 @@ Note:
 - `SIGKILL` cannot have a listener installed, it will unconditionally terminate
   Node.js on all platforms.
 - `SIGSTOP` cannot have a listener installed.
-
-Note that Windows does not support sending Signals, but Node.js offers some
-emulation with `process.kill()`, and `child_process.kill()`:
-- Sending signal `0` can be used to search for the existence of a process
```

Comment:
Why did you change this? What was the problem with the original?

You made the documentation wrong. Both comments refer ***specifically*** to Windows. Sending SIGINT or SIGTERM on non-windows does absolutely NOT cause the conditional exit of the target process.